s3/src/main/scala/geotrellis/spark/io/s3/S3AttributeStore.scala

```diff
@@ -98,7 +98,6 @@ class S3AttributeStore(val bucket: String, val prefix: String) extends BlobLayer
       .exists(_.getKey.endsWith(s"${AttributeStore.Fields.metadata}${SEP}${layerId.name}${SEP}${layerId.zoom}.json"))
 
   def delete(layerId: LayerId, attributeName: String): Unit = {
-    if(!layerExists(layerId)) throw new LayerNotFoundError(layerId)
```

Comment:
Why do we need to remove it here? O: 
All backends have this check, mb to make it more smart?